js/read_file.ts

```diff
@@ -42,5 +44,8 @@ export function readFileSync(filename: string): Uint8Array {
  *       console.log(decoder.decode(data));
  */
 export async function readFile(filename: string): Promise<Uint8Array> {
-  return res(await dispatch.sendAsync(...req(filename)));
+  const file = await open(filename);
+  const contents = await readAll(file);
+  file.close();
+  return contents;
```

Comment:
👍 

readFileSync should be similarly easy.

Reply:
Not really, I just realized that both `open` and `read` have only asynchronous... Is there sane way we can utilize promise to behave like blocking op?

Reply:
read should have a synchronous version... 

As for openSync - you'd need to add that. Shouldn't be too hard.

Reply:
I guess I can just add `openSync` and `readSync`...

Reply:
(oh looks like you're right. You need to add readSync too.)